### ISSUE TITLE
Forms Do Not Meet WCAG 2 Level A (1 post)

### ISSUE THREAD

**Issue by zackkatz** [katzwebservices/Constant-Contact-WordPress-Plugin#42]
> I could not find a form template currently available on Contact Contact which meets WCAG 2, Level A. The templates all appear to have label elements, but their for attributes all use the same ID which is not associated with an input element. Is there another template or set of fields that I missed which meets this requirement?

https://wordpress.org/support/topic/forms-do-not-meet-wcag-2-level-a
